packages/core/src/extensions/Placeholder/PlaceholderExtension.ts

```diff
@@ -13,54 +12,42 @@ const PLUGIN_KEY = new PluginKey(`blocknote-placeholder`);
  *
  */
 export interface PlaceholderOptions {
-  placeholder: Record<
-    string | "default" | "addBlock",
-    | string
-    | {
-        placeholder: string;
-        mustBeFocused: boolean;
-      }
-  >;
+  placeholders: Record<string | "default", string>;
 }
 
-export const Placeholder = Extension.create<PlaceholderOptions>({
+export const Placeholder = Extension.create<
+  PlaceholderOptions,
+  { styleEl: HTMLStyleElement }
+>({
   name: "placeholder",
 
+  addStorage() {
+    return {
+      styleEl: document.createElement("style"),
+    };
+  },
+
   addOptions() {
     return {
-      placeholder: {
+      placeholders: {
         default: "Enter text or type '/' for commands",
-        addBlock: "Type to filter",
-        heading: {
-          placeholder: "Heading",
-          mustBeFocused: false,
-        },
-        bulletListItem: {
-          placeholder: "List",
-          mustBeFocused: false,
-        },
-        numberedListItem: {
-          placeholder: "List",
-          mustBeFocused: false,
-        },
+        heading: "Heading",
+        bulletListItem: "List",
+        numberedListItem: "List",
       },
     };
   },
 
-  addProseMirrorPlugins() {
-    const styleEl = document.createElement("style");
-
-    // Append <style> element to <head>
+  onCreate() {
```

Comment:
can't we use the PM API for this instead of TipTap API? (we're trying this as a general rule right?)